app/models/local_authority.rb

```diff
@@ -5,4 +5,8 @@ class LocalAuthority < ApplicationRecord
   has_many :planning_applications, dependent: :destroy
 
   validates :council_code, presence: true
+
+  def signatory
+    "#{signatory_name}, #{signatory_job_title}"
+  end
```

Comment:
Not 100% sure about 
1. Whether a view helper would be more appropriate than a method in the model.
2. Whether this should be called `#signatory_name_and_job_title` instead!

Reply:
We could probably update the email text to use this method now. I think they all currently sign off with `@planning_application.local_authority.signatory_name %>, <%= @planning_application.local_authority.signatory_job_title`

Reply:
I've actually just updated them all to use `<%= @planning_application.local_authority.name %>` instead 😞  😆